plugin/kubernetes/kubernetes.go

```diff
@@ -396,12 +397,12 @@ func (k *Kubernetes) findPods(r recordRequest, zone string) (pods []msg.Service,
 		}
 
 		// exclude pods in the process of termination
-		if !p.ObjectMeta.DeletionTimestamp.IsZero() {
+		if !p.DeletionTimestamp.IsZero() {
```

Comment:
Should this check get moved to the object/pod.go object.topod conversion instead of saving the deletion timestamp?

Reply:
makes sense to me.

Reply:
done